pkgs/development/ocaml-modules/cmarkit/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, cmdliner
+, fetchurl
+, findlib
+, ocaml
+, ocamlbuild
+, topkg
+}:
+
+if lib.versionOlder ocaml.version "4.14.0"
+then throw "cmarkit is not available for OCaml ${ocaml.version}"
+else
+
+stdenv.mkDerivation rec {
+  pname = "cmarkit";
+  version = "0.1.0";
+
+  src = fetchurl {
+    url = "https://erratique.ch/software/cmarkit/releases/cmarkit-${version}.tbz";
+    hash = "sha256-pLPCLlwJt5W5R92HPY8gGpisyjlbSaaEe0HLuJlkjuY=";
+  };
+
+  nativeBuildInputs = [
+    ocaml
+    findlib
+    ocamlbuild
+    topkg
+  ];
+
+  buildInputs = [
+    topkg
+    cmdliner
+  ];
+
+  strictDeps = true;
+
+  inherit (topkg) buildPhase installPhase;
```

Comment:
why not enable the `cmdliner` build too? https://github.com/dbuenzli/cmarkit/blob/b20bdae5dbf3a4b50e150562eb8194a626d05271/opam#L50-L51

Reply:
The `bin` seems to be built either way?